client/src/App.tsx

```diff
@@ -19,7 +20,7 @@ const App = () => {
           <Route index element={<Home />}></Route>
```

Comment:
Should we change this to `<Route path="/" exact index element={<Home />}></Route>` ?

Reply:
On the **quick start** documentation section they have the following implementation:

```
import ReactDOM from "react-dom/client";
import {
  BrowserRouter,
  Routes,
  Route,
} from "react-router-dom";
// import your route components too

const root = ReactDOM.createRoot(
  document.getElementById("root")
);
root.render(
  <BrowserRouter>
    <Routes>
      <Route path="/" element={<App />}>
        <Route index element={<Home />} />
        <Route path="teams" element={<Teams />}>
          <Route path=":teamId" element={<Team />} />
          <Route path="new" element={<NewTeamForm />} />
          <Route index element={<LeagueStandings />} />
        </Route>
      </Route>
    </Routes>
  </BrowserRouter>
);
```

 [Explanation of the react router reserved word "index" or indexRoute](https://reactrouter.com/en/v6.3.0/getting-started/concepts#index-routes)
 
 What do you think would be better?

Reply:
Oh I like that better, lets use that example. We will probably have to fix this multiple times since we are going to be building lots of new stuff but at least its a good starting point. I just didn't like that we didn't have `path="/"` anywhere.